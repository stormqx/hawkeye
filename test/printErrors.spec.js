// import Listr from 'listr'
const makeConsoleMock = require('consolemock')
const printErrors = require('../src/printErrors')

describe('printErrors', () => {
  const originalConsole = console

  beforeAll(() => {
    console = makeConsoleMock()
  })

  beforeEach(() => {
    console.clearHistory()
  })

  afterAll(() => {
    global.console = originalConsole
  })

  it('should print plain errors', () => {
    const err = new Error('We have a problem')
    printErrors(err)
    expect(console.printHistory()).toMatchSnapshot()
  })
})
