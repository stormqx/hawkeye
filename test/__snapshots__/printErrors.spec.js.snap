// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`printErrors should print plain errors 1`] = `
"
ERROR We have a problem"
`;
